index.bs

```diff
@@ -11949,37 +11957,32 @@ the interfaces that |O| implements.
             and |prototype| has an own property named |P|, then return false.
         1.  Set |prototype| to be the value of the internal \[[Prototype]] property of |prototype|.
     1.  Return true.
-</div>
-
-<div class="note">
 
-    This should ensure that for objects with named properties, property resolution is done in the following order:
+    <div class="note">
 
-    1.  Indexed properties.
-    1.  Own properties, including unforgeable attributes and operations.
-    1.  Then, if [{{OverrideBuiltins}}]:
-        1.  Named properties.
-        1.  Properties from the prototype chain.
-    1.  Otherwise, if not [{{OverrideBuiltins}}]:
-        1.  Properties from the prototype chain.
-        1.  Named properties.
+        This should ensure that for objects with named properties, property resolution is done in the following order:
 
+        1.  Indexed properties.
+        1.  Own properties, including unforgeable attributes and operations.
+        1.  Then, if [{{OverrideBuiltins}}]:
+            1.  Named properties.
+            1.  Properties from the prototype chain.
+        1.  Otherwise, if not [{{OverrideBuiltins}}]:
+            1.  Properties from the prototype chain.
+            1.  Named properties.
+    </div>
 </div>
 
-Support for [=getters=] is
-handled by the <a href="#getownproperty">platform object \[[GetOwnProperty]] method</a>
-defined in section [[#getownproperty]], and
-for [=setters=]
-by the <a href="#defineownproperty">platform object \[[DefineOwnProperty]] method</a>
-defined in section [[#defineownproperty]] and the <a href="#platformobjectset">platform object \[[Set]] method</a>
-defined in section [[#platformobjectset]].
+Support for [=getters=] is handled by the <a href="#legacy-platform-object-getownproperty">legacy platform object \[[GetOwnProperty]] method</a>,
+and for [=setters=] by the <a href="#legacy-platform-object-defineownproperty">legacy platform object \[[DefineOwnProperty]] method</a>,
+and the <a href="#legacy-platform-object-set">legacy platform object \[[Set]] method</a>.
 
 
-<h4 id="getownproperty-guts" dfn>The PlatformObjectGetOwnProperty abstract operation</h4>
+<h4 id="getownproperty-guts" lt="LegacyPlatformObjectGetOwnProperty|The PlatformObjectGetOwnProperty abstract operation" dfn>The LegacyPlatformObjectGetOwnProperty abstract operation</h4>
```

Comment:
> Why the alternative lt=""

Because I didn't realise it wasn't exported (that dfn are unexported by default still confuses me). Fixing.

> Also, you should probably add export in case it needs to be mentioned somewhere.

So the way these legacy platform operations are exported seems very ad hoc. Do we want to consistently export all of them, or wait until someone requests access?